### PR TITLE
Add components

### PR DESCRIPTION
**Pull request by Abe1able**
### In this milestone of the project, I have

- initialized the project with react components
- used react router for navigation.
- made sure there are no linter errors.

### CODE REVIEW

**Comment by codecaiine** [comment]
Hi @Abe1able can you go on slack ?

**Comment by codecaiine** [comment]
Hi @Abe1able I left a message on slack

File: src/components/Categories.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>
+    <h2>Under construction</h2>
```
Comment (codecaiine) [inline src/components/Categories.js]: - [x] Good job so far :+1: But according to the project requirement the Categories should display a button "Check status" only. Please fix it in order to make your project complete and works as expected :+1:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Abe1able , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Great project implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Categories.js (hunk 1 above)
Comment (Abe1able) [reply src/components/Categories.js]: Fixed, thank you.

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @Abe1able, 

To highlight 👍🏽 
✔️ No linter errors
✔️ Professional README

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**